### ISSUE TITLE
Creating own dataset

### ISSUE THREAD

**Issue by Wild1234** [devicehive/devicehive-audio-analysis#8]
Good night.

I want to create my own dataset with my own labels. is it possible for this repository?

Thanks

**Comment by igor-panteleev** [comment]
Hi,
Yes, you can use your own dataset.

You need to extract features from raw data.
To do this take a look to vggish lib [here](https://github.com/tensorflow/models/tree/master/research/audioset).

Also you can find [how i did this on the fly](https://github.com/devicehive/devicehive-audio-analysis/blob/master/audio/processor.py#L86-L88).

Hope it will help.

**Comment by Lelo123** [comment]
Thanks!

But isn't necessary to train a model with youtube 8M?

**Comment by igor-panteleev** [comment]
Short answer - yes.
Two models have been used here.
vggish - as feature extractor.
youtube8m - as classifier.

So if you want to use different labels you need to extract features using vggish and then train youtube 8m model with these features.

**Comment by hei9gag** [comment]
Hi,

I still have some questions about creating own data set. 

The training script provided by youtube8m is using .tfrecord extension. Do you know how to generate this format for audio?

Also, how do I add my custom labels using the youtube8m training model? Many thanks.
